index.html

```diff
@@ -5,15 +5,36 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>TV Show Project | Leida Goncalves (Leidagandy)</title>
     <link href="style.css" rel="stylesheet" />
-    <link rel="icon" href="/favicon.ico" type="image/x-icon" />
+    <!-- <link rel="icon" href="/favicon.ico" type="image/x-icon" /> -->
+    <link rel="icon" href="./favicon.ico" type="image/x-icon" />
   </head>
 
   <body>
+    <header>
+      <div id="page-header">
+        <select id="selectedEpisode"
+          >Select an episode</select
+        >
+        <form>
+          <input
+            id="searchEpisode"
+            type="text"
+            placeholder="Search episodes..."
+          />
+        </form>
+        <span id="searchResult">Displaying 73/73 episodes</span>
+      </div>
+    </header>
     <div id="root"></div>
 
+    <button id="viewAllEpisodes">View all episodes</button>
```

Comment:
Good use of IDs through your HTML so far. We use IDs when elements are unique, and I can see you have done that here :) good job!